src/olympia/addons/serializers.py

```diff
@@ -242,6 +244,12 @@ def get_categories(self, obj):
     def get_has_eula(self, obj):
         return bool(getattr(obj, 'has_eula', obj.eula))
 
+    def get_is_featured(self, obj):
+        if not hasattr(obj, '_is_featured'):
+            # Any featuring will do.
+            obj._is_featured = obj.is_featured(app=None, lang=None)
```

Comment:
I tried to follow the pattern of the other serializer methods that rely on ES if present, like `has_eula`, but can do. I was of two minds if I should just override the `def get_is_featured` method in the ES serializer subclass instead, but it was inconsistent.